[BUG] `test_read_compressed_hive_text` fails on CDH

`test_read_compressed_hive_text` fails on CDH, thus:
```
[2023-06-01T13:21:06.888Z] =================================== FAILURES ===================================
[2023-06-01T13:21:06.888Z] [31m[1m__________________ test_read_compressed_hive_text[BZip2Codec] __________________[0m
[2023-06-01T13:21:06.888Z] 
[2023-06-01T13:21:06.888Z] spark_tmp_table_factory = <conftest.TmpTableFactory object at 0x7fb56374cee0>
[2023-06-01T13:21:06.888Z] codec = 'BZip2Codec'
[2023-06-01T13:21:06.888Z] 
[2023-06-01T13:21:06.888Z]     @pytest.mark.parametrize('codec', ['BZip2Codec',    # BZ2 compression, i.e. Splittable.
[2023-06-01T13:21:06.888Z]                                        'DefaultCodec',  # DEFLATE, i.e. Gzip, without headers. Unsplittable.
[2023-06-01T13:21:06.888Z]                                        'GzipCodec'])    # Gzip proper. Unsplittable.
...
[2023-06-01T13:21:06.888Z] [1m[31mE               pyspark.sql.utils.IllegalArgumentException: Part of the plan is not columnar class org.apache.spark.sql.hive.execution.HiveTableScanExec[0m
[2023-06-01T13:21:06.888Z] [1m[31mE               Scan hive default.tmp_table_main_1771630401_0 HiveTableRelation [`default`.`tmp_table_main_1771630401_0`, org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe, Data Cols: [id#19L, id_string#20], Partition Cols: []][0m
```

It shouldn't even be running on CDH, because of https://github.com/NVIDIA/spark-rapids/issues/7423.

--- a/integration_tests/src/main/python/hive_delimited_text_test.py
+++ b/integration_tests/src/main/python/hive_delimited_text_test.py
@@ -422,6 +422,9 @@ def test_custom_timestamp_formats_disabled(spark_tmp_path, data_gen, spark_tmp_t
         conf=hive_text_enabled_conf)
 
 
+@pytest.mark.skipif(is_spark_cdh(),
+                    reason="Hive text reads are disabled on CDH, as per "
+                           "https://github.com/NVIDIA/spark-rapids/pull/7628")
 @pytest.mark.parametrize('codec', ['BZip2Codec',    # BZ2 compression, i.e. Splittable.
                                    'DefaultCodec',  # DEFLATE, i.e. Gzip, without headers. Unsplittable.
                                    'GzipCodec'])    # Gzip proper. Unsplittable.

